frontend/src/pages/accounts/profile.module.scss

```diff
@@ -71,12 +71,60 @@
         display: flex;
       }
 
+      .stat-wrapper {
+        position: relative;
+
+        $arrowWidth: 6px;
+
+        .tooltip {
+          @include text-body-sm;
+          position: absolute;
+          width: $content-xs;
+          color: $color-white;
+          max-width: calc(100% - 2 * $spacing-xs);
+          top: calc(100% + $arrowWidth + $spacing-xs);
+          right: $spacing-xs;
+          background-color: $color-purple-80;
+          box-shadow: $box-shadow-sm;
+          display: flex;
+          flex-direction: column;
+          gap: $spacing-sm;
+          padding: $spacing-md $spacing-md;
+          border-radius: $border-radius-sm;
+          // Prevent other .stat-wrapper elements from overlapping this tooltip:
+          z-index: 2;
+
+          @media screen and #{$mq-lg} {
+            // The wrapper doesn't span the full width on large screens,
+            // so limiting the tooltip to its wrapper's size makes it too thin:
+            max-width: unset;
+          }
+
+          &::before {
+            content: "";
+            height: 2 * $arrowWidth;
+            width: 2 * $arrowWidth;
+            position: absolute;
+            top: -1 * $arrowWidth;
```

Comment:
Question: Wouldn’t these need to be wrapped in `calc()`?

Reply:
This calculation can be done at build time (`$arrowWidth` is a Sass variable, and thus never changes at runtime), so Sass will do it at build time. In other words, this is SCSS syntax, not CSS :)